packages/back-end/src/controllers/reports.ts

```diff
@@ -192,10 +192,13 @@ export async function refreshReport(
 
   const useCache = !req.query["force"];
 
-  report.args.statsEngine =
-    report.args?.statsEngine || org.settings?.statsEngine || "bayesian";
-  report.args.regressionAdjustmentEnabled = !!report.args
-    ?.regressionAdjustmentEnabled;
+  const statsEngine = report.args?.statsEngine || "bayesian";
+
+  report.args.statsEngine = statsEngine;
+  report.args.regressionAdjustmentEnabled =
+    statsEngine === "frequentist"
+      ? !!report.args?.regressionAdjustmentEnabled
+      : false;
```

Comment:
Can we be 'dumb' about passing in `regressionAdjustmentEnabled`?

```
  report.args.statsEngine = statsEngine;
  report.args.regressionAdjustmentEnabled = report.args?.regressionAdjustmentEnabled;
```

And trust that `getScopedSettings` should take care of appropriate validation/rectification later?

Reply:
I'd prefer to only directly influence statsEngine in this PR, not dependent fields like `regressionAdjustmentEnabled`. Those need more work and testing to make sure we've gotten it right.